
Disable xfs in grub-probe.  xfs support is still experimental and buggy, so we
can't rely on grub-probe succesfuly identifiing an xfs filesystem just to get a
boot failure later.

--- util/grub-probe.c~	2007-12-23 19:12:24.000000000 +0100
+++ util/grub-probe.c	2007-12-23 19:12:07.000000000 +0100
@@ -126,6 +126,9 @@
   if (! fs)
     grub_util_error ("%s", grub_errmsg);
 
+  if (strcmp (fs->name, "xfs") == 0)
+    grub_util_error ("xfs detection disabled in Debian package");
+
   printf ("%s\n", fs->name);
   
   grub_device_close (dev);
